upgrade.json

```diff
@@ -31,11 +31,12 @@
               "20151223-edit-note-index.sql",
               "20160310-mbs-4501-alternative-tracklists.sql",
               "20160516-mbs-8656.sql",
-              "20160516-mbs-8804.sql",
               "20160507-mbs-8727.sql",
               "20160512-edit-data.sql",
               "20160514-mbs-8287-a.sql",
-              "20160514-mbs-8287-b.sql"],
+              "20160514-mbs-8287-b.sql",
+              "20160515-recluster.sql",
+              "20160516-mbs-8804.sql"],
```

Comment:
As I said above, reclustering should happen before creating the BRIN indexes in `20160516-mbs-8804.sql`.